tests/unit/sproxyd.js

```diff
@@ -54,6 +54,23 @@ function _batchDelKeys(n) {
     return list;
 }
 
+function checkKeyContent(client, key, body, reqUid, callback) {
+    client.get(key, undefined, reqUid, (err, stream) => {
+        let ret = Buffer.alloc(0);
+        if (err) {
+            done(err);
+        } else {
+            stream.on('data', val => {
+                ret = Buffer.concat([ret, val]);
```

Comment:
It doesn't really matter here as it's for unit tests, but in general it's more efficient to
1. build a list of chunks (from `const chunks = []`), appending for every data chunk received in `'data'` events (`chunks.push(val)`)
2. concatenate all chunks in a single buffer, with `ret = Buffer.concat(chunks)` in the `'end'` event handler

Reply:
It's basically to avoid re-constructing a new buffer each time we append a small chunk to it, which causes the entire buffer to be copied each time as we rebuild the buffer every time we call `Buffer.concat` (so a time complexity of O(n2) wrt. the final size of the buffer).